presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1180,7 +1180,7 @@ private void testColumnName(String columnName, boolean delimited)
 
         try {
             // TODO test with both CTAS *and* CREATE TABLE + INSERT, since they use different connector API methods.
-            assertUpdate("CREATE TABLE " + tableName + "(id varchar, " + nameInSql + " varchar)");
+            assertUpdate("CREATE TABLE " + tableName + "(key varchar, " + nameInSql + " varchar)");
```

Comment:
> Rename id column to key in testColumnName

in the commit message maybe mention that `id` has a special meaning in Cassandra.